src/include/txn_inline.h

```diff
@@ -471,7 +574,7 @@ __wt_txn_op_set_timestamp(WT_SESSION_IMPL *session, WT_TXN_OP *op)
     txn = session->txn;
 
     if (!__txn_should_assign_timestamp(session, op))
-        return;
+        return (0);
```

Comment:
Do we not need to check for timestamps in the if branch (`if (F_ISSET(txn, WT_TXN_PREPARE))`)?

Reply:
Ops sorry my bad, we only check for commit.